.husky/pre-commit

```diff
@@ -15,4 +15,12 @@ for file in $STAGED_FILES; do
     fi
 done
 
-exit 0
\ No newline at end of file
+# Prevent commiting .npmrc with the registry set to local.
+HAS_LOCAL_REGISTRY=$(git diff --cached --name-only .npmrc | xargs grep -n "registry=http://localhost:4873/")
+if [ -n "${HAS_LOCAL_REGISTRY}" ]; then
+	echo "The commit was rejected due to attempting to commit .npmrc with the registry set to the local registry."
```

Comment:
Is it possible to use a file that's `.gitignore`-d instead of a versioned one like `.npmrc`?

Reply:
Alternatively, could this focus more on the next step? As in tell the user to run `npm run local-registry:stop`